rundeckapp/grails-app/views/framework/editProject.gsp

```diff
@@ -90,7 +90,6 @@
     <asset:javascript src="leavePageConfirm.js"/>
     <asset:javascript src="framework/editProject.js"/>
     <asset:javascript src="static/pages/project-config.js" defer="defer" />
-    <asset:stylesheet href="static/css/pages/project-config.css" />
```

Comment:
This file isn't being created, as the respective components do not have any css to be imported.